src/firewall/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.1.4 on 2023-04-15 23:21
+# Generated by Django 4.2 on 2023-04-16 08:38
```

Comment:
It appears here that you are using a different version of Django than the one defined on [requirements.txt#L4](https://github.com/IPv18/NOCD/blob/main/requirements.txt#L4)
Please consider switching to "Django==4.1.4". 
Or better, create a venv with [requirements.txt](https://github.com/IPv18/NOCD/blob/main/requirements.txt)

Reply:
resolved